src/main/java/com/marklogic/appdeployer/command/schemas/LoadSchemasCommand.java

```diff
@@ -0,0 +1,51 @@
+package com.marklogic.appdeployer.command.schemas;
+
+import java.io.File;
+
+import com.marklogic.appdeployer.AppConfig;
+import com.marklogic.appdeployer.command.AbstractCommand;
+import com.marklogic.appdeployer.command.CommandContext;
+import com.marklogic.appdeployer.command.SortOrderConstants;
+import com.marklogic.client.DatabaseClient;
+import com.marklogic.client.schemasloader.SchemasLoader;
+import com.marklogic.client.schemasloader.impl.DefaultSchemasFinder;
+import com.marklogic.client.schemasloader.impl.DefaultSchemasLoader;
+
+public class LoadSchemasCommand extends AbstractCommand  {
+
+	private SchemasLoader schemasLoader;
+	
+	public LoadSchemasCommand() {
+        setExecuteSortOrder(SortOrderConstants.LOAD_SCHEMAS);
+	}
+	
+	@Override
+	public void execute(CommandContext context) {
+		loadSchemasIntoSchemasDatabase(context);
+	}
+
+	protected void loadSchemasIntoSchemasDatabase(CommandContext context) {
+		if (schemasLoader == null) {
+			initializeDefaultSchemasLoader(context);
+		}
+		
+		
+		AppConfig config = context.getAppConfig();
+        DatabaseClient client = config.newSchemasDatabaseClient();
+        try {
+        	File baseDir = config.getConfigDir().getBaseDir();
```

Comment:
This is the line I was referring to in the ml-javaclient-util PR - this command is looking for the schemas under src/main/ml-config/schemas. If you think that's fairly intuitive - if schemas are typically considered part of the "config" of an app vs the "code" of an app - I'm good with it. 

One potential downside is that it means everything under src/main/ml-config except schemas is loaded via the Management API, with schemas being loaded via the Client API. 

Maybe it should be "src/main/ml-schemas"?? I don't really know.